README.md

```diff
@@ -1,4 +1,4 @@
-<img align="left" width="64" height="64" src="https://raw.githubusercontent.com/starfish-app/starfish/main/data/icons/64.svg">
+<img align="left" width="64" height="64" src="https://raw.githubusercontent.com/starfish-app/starfish/main/data/icons/64/hr.from.josipantolis.starfish.svg">
```

Comment:
I'd prefer not to have project name repeated in too many places, such as icon file names, etc.
Ideally it would only be defined in the `project` function call on top of the root meson.build file.
Individual files can be renamed in calls to `install_data` function. That approach allows for:

1. Shorter file names without repetition. This just makes working with them more comfortable.
2. Easier rename of the project in the future. Either if someone wanted to maintain a fork long term, or just easily install 2 versions on the same system.

Admittedly, I haven't followed this rule in all the places myself, but I plan to clean those up over time.